server/src/main/java/org/apache/iotdb/db/protocol/influxdb/meta/TagInfoRecords.java

```diff
@@ -65,11 +66,8 @@ public TagInfoRecords() {
 
   public void add(String database, String measurement, String tag, int order) {
     deviceIds.add(TAG_INFO_DEVICE_ID);
-    if (times.size() == 0) {
-      times.add(System.currentTimeMillis());
-    } else {
-      times.add(times.get(times.size() - 1) + 1);
-    }
+    // Multiple adjacent records, possibly with the same timestamp
+    times.add(System.currentTimeMillis());
```

Comment:
Hi, nice work! I have a little confuse about this bug fix. Since the value of this timestamp just be used to record the order of inserting, why not use AtomicLong and its getAndIncrement method to generate the timestamp?

Reply:
Your idea is indeed feasible, you can first take a look at https://github.com/apache/iotdb/pull/6757, this is @jun0315 recently merged code, he uses the system time here.

Reply:
When I submitted the code and merged the conflict, I forgot to resolve the conflict and used my code directly. How to solve the bug here, what are @jun0315  thoughts?

Reply:
@LIU-WEI-git Thanks for your review, I think your idea is very good. If we consider the distributed situation, using AtomicLong is a good solution.
@KeeProMise I also support use AtomicLong and its getAndIncrement method.

Reply:
ok , it has been modified to use the AtomicLong to generate the tag timestamps

Reply:
@KeeProMise I saw your latest commit. Nice work!